src/RpcClient/Models/RpcNep5Transfers.cs

```diff
@@ -0,0 +1,81 @@
+using Neo.IO.Json;
+using Neo.SmartContract;
+using Neo.Wallets;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+
+namespace Neo.Network.RPC.Models
+{
+    public class RpcNep5Transfers
+    {
+        public UInt160 UserScriptHash { get; set; }
+
+        public List<RpcNep5Transfer> Sent { get; set; }
+
+        public List<RpcNep5Transfer> Received { get; set; }
+
+        public JObject ToJson()
+        {
+            JObject json = new JObject();
+            json["sent"] = Sent.Select(p => p.ToJson()).ToArray();
+            json["received"] = Received.Select(p => p.ToJson()).ToArray();
+            json["address"] = UserScriptHash.ToAddress();
+            return json;
+        }
+
+        public static RpcNep5Transfers FromJson(JObject json)
+        {
+            RpcNep5Transfers transfers = new RpcNep5Transfers
+            {
+                Sent = ((JArray)json["sent"]).Select(p => RpcNep5Transfer.FromJson(p)).ToList(),
+                Received = ((JArray)json["received"]).Select(p => RpcNep5Transfer.FromJson(p)).ToList(),
+                UserScriptHash = json["address"].AsString().ToScriptHash()
+            };
+            return transfers;
+        }
+    }
+
+    public class RpcNep5Transfer
+    {
+        public ulong TimestampMS { get; set; }
+
+        public UInt160 AssetScriptHash { get; set; }
+
+        public UInt160 UserScriptHash { get; set; }
+
+        public BigInteger Amount { get; set; }
+
+        public uint BlockIndex { get; set; }
+
+        public ushort TransferNotifyIndex { get; set; }
+
+        public UInt256 TxHash { get; set; }
+
+        public JObject ToJson()
+        {
+            JObject json = new JObject();
+            json["timestamp"] = TimestampMS;
+            json["asset_hash"] = AssetScriptHash.ToString();
```

Comment:
Camelcase for all? `assetHash`?

Reply:
Hi @shargon,
Do you mean also change "asset_hash" in Plugin `RpcNep5Tracker`?

Reply:
I mean, Ussually we use this format `assetHash` instead of `asset_hash` isn't it ?

Reply:
Yes, how about move these models to `RpcServer`, then we can use the `ToJson()` method in all the plugins.

Reply:
Hi @shargon, I have move models to `RpcServer`. Camelcase for all will change the JSON in several responses, I will do the change if you think it's necessary.

Reply:
No....... These models are not belong to `RpcServer`...

Reply:
OK, I have move models back😂